share/revng/test/configuration/revng-qa/architectures.yml

```diff
@@ -46,15 +46,20 @@ tags:
         - -Wl,-Ttext-segment=0x20000
       COMMON_CFLAGS:
         - -DTARGET_arm
+      CLANG_CFLAGS:
+        - --target=arm
```

Comment:
If you want to do this, use `"$TRIPLE"` as `--target`. There might be subtle differences, let's use the same triple.

Alternatively, we can have `clang-release` install `armv7a-hardfloat-linux-uclibceabi-clang` symlinked to `clang`, which should be equivalent to `clang --target armv7a-hardfloat-linux-uclibceabi`. This is the appropriate solution, but I'm OK with `--triple` if you leave a `TODO`.

Reply:
Left a todo.